.github/workflows/deploy-argocd.yml

```
@@ -23,4 +23,4 @@ jobs:
         if: ${{ steps.manifest.outputs.MQ_HAS_OUTPUT == 'true' }}
         with:
           files: ${{ steps.manifest.outputs.MQ_CHANGED_FILES }}
-          gh_token_pat: ${{ secrets.GH_HELM_VALUES_PAT }}
+          ssh-key: ${{ secrets.HELM_VALUES_SSH_KEY }}
```

actionlint (1.7.4)

input "ssh-key" is not defined in action "Update helm-values charts" defined at "./.github/actions/update-helm-values". available inputs are "files", "gh_token_pat"

(action)